create-security-certificates.md

```diff
@@ -1,103 +1,199 @@
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [command](cockroach-commands.html) with the appropriate subcommands and flags.
 
 When using <code>cockroach cert</code> to create node and client certificates, you will need access to a local copy of the CA certificate and key. It is therefore recommended to create all certificates and keys in one place and then distribute node and client certificates and keys appropriately. For the CA key, be sure to store it somewhere safe and keep a backup; if you lose it, you will not be able to add new nodes or clients to your cluster. For a walkthrough of this process, see [Manual Deployment](manual-deployment.html).
 
 <div id="toc"></div>
 
 ## Subcommands
 
-Subcommand | Usage 
+Subcommand | Usage
 -----------|------
-`create-ca` | Create the self-signed certificate authority (CA), which you'll use to create and authenticate certificates and keys for your entire cluster.
+`create-ca` | Create the self-signed certificate authority (CA), which you'll use to create and authenticate certificates for your entire cluster.
 `create-node` | Create a certificate and key for a specific node in the cluster. You specify all addresses at which the node can be reached and pass appropriate flags.
 `create-client` | Create a certificate and key for a [specific user](create-and-manage-users.html) accessing the cluster from a client. You specify the username of the user who will use the certificate and pass appropriate flags.
+`list` | List certificates and keys found in the certificate directory.
+
+## Certificate Directory
+
+The `create-*` subcommands generate the CA certificate and all node and client certificates and keys in a single directory specified by the `--certs-dir` flag and names the files as follows:
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate
+`node.key`   | Key for server certificate
+`client.<user>.crt` | Client certificate for `<user>` (eg: `client.root.crt` for user `root`)
+`client.<user>.key` | Key for the client certificate
+
+Note the following:
+
+- The CA key is never loaded automatically by `cockroach` commands, so it should be created in a seperate directory, identified by the `--ca-key` flag.
+
+- Keys (files ending in `.key`) must not have group or world permissions (maximum permissions are 0700, or `rwx------`). This check can be disabled by setting the environment variable `COCKROACH_SKIP_KEY_PERMISSION_CHECK=false`.
```

Comment:
oops, my bad, this is supposed to say `COCKROACH_SKIP_KEY_PERMISSION_CHECK=true`